src/tiles/FeedTile.vue

```diff
@@ -15,20 +15,20 @@
 </template>
 
 <script>
-    import * as marquee from './../marquee'
+    import { Marquee } from './../marquee'
```

Comment:
Quick note: If you'll make `Marquee` a `default` export, then you'd want to remove this braces.